### PR TITLE
noticed that in higher dispute rounds all of the users rep wasn't use…

### PR DESCRIPTION
**Pull request by bthaile**
Needed when not sending in amount just use accounts full balance.

### CODE REVIEW

File: scripts/flash/dispute-contribute.js [hunk 1]
```diff
@@ -45,76 +44,96 @@ function disputeContribute(augur, args, auth, callback) {
...
+              console.log(chalk.red("feeWindowId has not been created"));
+              return callback("Market doesn't have fee window, need to report");
+            }
+            console.log(chalk.yellow("Market Fee Window"), chalk.yellow(feeWindowId));
+            var feeWindowPayload = { tx: { to: feeWindowId } };
+            augur.api.FeeWindow.getStartTime(feeWindowPayload, function (err, feeWindowStartTime) {
               if (err) {
                 console.log(chalk.red(err));
-                return callback(err);
+                callback("Could not get Fee Window");
```
Comment (adrake33) [inline scripts/flash/dispute-contribute.js]: I think this should be `return callback`

File: scripts/flash/dispute-contribute.js [hunk 2]
```diff
@@ -45,76 +44,96 @@ function disputeContribute(augur, args, auth, callback) {
...
+                        if (err) {
+                          return callback("Market contribute Failed");
+                        }
+
+                        console.log(chalk.green("Market contribute Done"));
+                        callback(null);
+                      });
+                    } else {
+                      console.log(chalk.red("Fee Window isn't active"));
+                      callback("Fee Window isn't active");
```
Comment (adrake33) [inline scripts/flash/dispute-contribute.js]: This should also probably be `return callback`.

File: scripts/flash/dispute-contribute.js (hunk 1 above)
Comment (bthaile) [reply scripts/flash/dispute-contribute.js]: good call

File: scripts/flash/dispute-contribute.js (hunk 2 above)
Comment (bthaile) [reply scripts/flash/dispute-contribute.js]: just for convention, I agree. will fix

**Pull request merged**